packages/runner/src/lib/event-manager.js

```diff
@@ -32,6 +32,12 @@ const socketRerunEvents = 'runner:restart watched:file:changed'.split(' ')
 const localBus = new EventEmitter()
 const reporterBus = new EventEmitter()
 
+if (window.Cypress) {
+  window.channel = ws
```

Comment:
let's add a note reflecting on why this is a weird shape - that we only want to expose this within the test environment, but because we use an e2e test that uses it, we either need to move that test to isolated-runner or find a 2nd way to expand the conditional.